environment.yml

```diff
@@ -9,8 +9,14 @@ dependencies:
   - pytest>=7.1
   - ruamel_yaml>=0.15
   - scikit-image>=0.18
+  - dask-image>=2024.5.3
+  - dask>= 2024.6.2
```

Comment:
can we add a note in the doc for if you have a numpy error it can be cause by numpy 2.0 or somthing